packages/features/lend/src/lend-fixed/hooks/use-fixed-liquidity-pools-table.tsx

```diff
@@ -0,0 +1,107 @@
+import { useContext } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { LendFixedContext } from '../../lend-fixed/lend-fixed';
+import { useFCashMarket } from '@notional-finance/notionable-hooks';
+import { DisplayCell, DataTableColumn } from '@notional-finance/mui';
+import { getNowSeconds } from '@notional-finance/util';
+import { RATE_PRECISION } from '@notional-finance/util';
+import {
+  getDateString,
+  formatNumber,
+  formatNumberAsPercent,
+} from '@notional-finance/helpers';
+
+export const useFixedLiquidityPoolsTable = (
```

Comment:
Isn't this the same code as the other hook? Why not just pass in the currencyId instead of using the context and all that and put this in the `shared/trade` package?